src/industry_gui.cpp

```diff
@@ -1303,7 +1304,10 @@ static bool CDECL CargoFilter(const Industry * const *industry, const std::pair<
 
 static GUIIndustryList::FilterFunction * const _filter_funcs[] = { &CargoFilter };
 
-
+/** Enum referring to the Hotkeys in the industry directory window */
+enum IndustryDirectoryHotkeys{
```

Comment:
```suggestion
enum IndustryDirectoryHotkeys {
```
Our coding style is to always have a space before `{`. See https://github.com/OpenTTD/OpenTTD/blob/master/CODINGSTYLE.md for the details.